am/config.py

```diff
@@ -103,3 +103,19 @@ def acoustic_model_config(
         config.phonology.future_length = phon_future_length
 
     return config
+
+
+def adjust_am_config_for_corrected_applicator(config: rasr.RasrConfig):
+    """
+    Set the correct type of applicator, default is "legacy". Moreover, set exit penalities to zero
+    For a given word sequence the exit penalty is constant with respect to the max/sum
```

Comment:
In case of optional silence, the silence exit penalty does make a difference, right?

Anyway, I would rather not set the exit TDP to 0.0 here, because then they might be different from what the user is expecting when he specifies other values in the crp. 
If you think they absolutely always should be zero, I would propose an assert statement rather than a silent overwrite.

Reply:
> In case of optional silence, the silence exit penalty does make a difference, right?

This should be defined on the lemma level for both silence and speech with proper normalization. For now we assume that optional silence and speech on lemma level get probability 1.0, so no penalty. The only transition weight is then forward/loop.

> Anyway, I would rather not set the exit TDP to 0.0 here, because then they might be different from what the user is expecting when he specifies other values in the crp.
Here we are only touching a copy of the crp for specific alignment job and not modifying the original crp.

Reply:
> > In case of optional silence, the silence exit penalty does make a difference, right?
>
> This should be defined on the lemma level for both silence and speech with proper normalization. 

I do not disagree with that, however the proper support to define this on lemma level does not exist yet, right?

> For now we assume that optional silence and speech on lemma level get probability 1.0, so no penalty. The only transition weight is then forward/loop.

Then this assumption would limit us in practice as it removes one degree of freedom.

> Here we are only touching a copy of the crp for specific alignment job and not modifying the original crp.

Ok, but same argument for this specific job.